packages/playwright-test/src/runner/reporters.ts

```diff
@@ -44,7 +44,10 @@ export async function createReporter(config: FullConfigInternal, mode: 'list' |
     blob: BlobReporter,
   };
   const reporters: Reporter[] = [];
-  for (const r of config.config.reporter) {
+  const descriptions: ReporterDescription[] = reporterNames ?
```

Comment:
What about something like this at line 51.5?

```
if (reporterNames && !reporterNames.includes(name)) continue;
```

Reply:
This won't work, some of `reporterNames` may not be in the config at all and we still want to create corresponding reports.